components/text_normalization/requirements.txt

```
@@ -0,0 +1,3 @@
+git+https://github.com/ml6team/fondant.git@main
```

```suggestion
fondant
```
As there was a new Fondant release, this should work

---

We should keep the install form git, since our build and release pipelines expect it.

See https://github.com/ml6team/fondant/pull/238.

---

Ok, will close this comment